site/realm/functions/onNewUser.js

```diff
@@ -0,0 +1,10 @@
+exports = async (authEvent) => {
+
+  const mongodb = context.services.get("mongodb-atlas");
+  const users = mongodb.db("customData").collection("users");
```

Comment:
Realm creates this `customData` collection? Does this PR require MongoDB changes?